packages/mobile/e2e/signUp.test.ts

```diff
@@ -26,9 +27,9 @@ describe('Sign up', () => {
     await device.launchApp()
   })
 
-  afterEach(async () => {
-    await device.reloadReactNative()
-  })
+  // afterEach(async () => {
```

Comment:
Is this not needed?

Reply:
ah good catch, it is, this was to make debugging easier for me